src/test/java/org/mule/extension/db/integration/model/DerbyTestDatabase.java

```diff
@@ -135,6 +135,13 @@
           "DYNAMIC RESULT SETS 0\n" +
           "EXTERNAL NAME '" + DerbyTestStoredProcedure.class.getName() + ".addOne'";
 
+  public static final String SQL_CREATE_SP_ADD_ONE_NO_SCHEMA =
```

Comment:
This SP will have a schema. When you don't specify a Derby uses the default Schema